From 891bdfcf2a7c65ac5d6e4beb54820e816240f2d7 Mon Sep 17 00:00:00 2001
From: Chris Renshaw <osm0sis@outlook.com>
Date: Sat, 9 May 2020 06:53:32 -0300
Subject: [PATCH 4/8] init: add vendor.* keys to spoof safetynet

aswinas@pixysos: add some more props from magisk hide to userspace hack by arter97

Former-commit-id: fc79269db601c9cd0dad3781d4e6ee8f209c55fc
---
 init/property_service.cpp | 8 ++++++++
 1 file changed, 8 insertions(+)

diff --git a/init/property_service.cpp b/init/property_service.cpp
index a9458ab..49cdb62 100644
--- a/init/property_service.cpp
+++ b/init/property_service.cpp
@@ -787,6 +787,10 @@ static const char *snet_prop_key[] = {
 	"ro.build.keys",
 	"ro.build.tags",
 	"ro.system.build.tags",
+	"ro.vendor.boot.warranty_bit",
+	"ro.vendor.warranty_bit",
+	"vendor.boot.vbmeta.device_state",
+	"vendor.boot.verifiedbootstate",
 	NULL
 };
 
@@ -804,6 +808,10 @@ static const char *snet_prop_value[] = {
 	"release-keys", // ro.build.keys
 	"release-keys", // ro.build.tags
 	"release-keys", // ro.system.build.tags
+	"0", // ro.vendor.boot.warranty_bit
+	"0", // ro.vendor.warranty_bit
+	"locked", // vendor.boot.vbmeta.device_state
+	"green", // vendor.boot.verifiedbootstate
 	NULL
 };
 
-- 
2.30.1

